packages/api/src/command/medical/patient/consolidated-create.ts

```diff
@@ -83,15 +82,17 @@ const convertCollectionBundleToTransactionBundle = ({
     )
       continue;
 
+    const contained = resource.contained ? [...resource.contained, patient] : [patient];
     const transactionEntry: BundleEntry = {
       resource: {
         ...resource,
-        contained: resource.contained ? [...resource.contained, patient] : [patient],
+        contained,
       },
-      request: resource.id
-        ? { method: "PUT", url: resource.resourceType + "/" + resource.id }
-        : { method: "POST", url: resource.resourceType },
+      request: { method: "PUT", url: resource.resourceType + "/" + resource.id },
     };
+    if (resource.resourceType === "Patient") {
+      delete (transactionEntry.resource as Patient).contained;
+    }
```

Comment:
Probably better to just never add contained to the object above if its `Patient`rather than add it and then delete it

Reply:
I was thinking that too, but that flow breaks the type checking. It really sucks..
Basically, I building an object with all the attributes besides `contained`, and then tried to check if it's `!== "Patient"`, then add the `contained`. But TS starts complaining because there's a bunch of other resourceTypes that can't have the `contained` attribute, and it becomes a wild mess

Reply:
I was able to do this and it seems to pass the TS

```

    const transactionEntry: BundleEntry = {
      resource,
      request: resource.id
        ? { method: "PUT", url: resource.resourceType + "/" + resource.id }
        : { method: "POST", url: resource.resourceType },
    };

    if (resource.resourceType !== "Patient") {
      transactionEntry.resource = {
        ...resource,
        contained: resource.contained ? [...resource.contained, patient] : [patient],
      };
    }
```

Reply:
Ah, that's good. Updated the code. 

I think I was trying 
```
    if (resource.resourceType !== "Patient") {
      transactionEntry.resource.contained = contained;
    }
```

in which case it errors with:
```
Property 'contained' does not exist on type 'Resource'.
  Property 'contained' does not exist on type 'AccessPolicy'
```